packages/server/src/api/routes/tests/search.spec.ts

```diff
@@ -156,6 +171,225 @@ describe.each([
     return expectSearch({ query })
   }
 
+  // Ensure all bindings resolve and perform as expected
+  !isSqs &&
```

Comment:
How come you're excluding SQS here? Is it because it _doesn't_ work, or is it because it _shouldn't_ work?

Reply:
The tests started failing at the SQS stage so I added the exclusion to verify that the rest of the matrix was running without issue. I'll refactor out my snippet changes and try to figure out what the issue is with SQS.

Reply:
Don't worry about it, leave this in and put a `TODO(samwho): fix with SQS` or some such. I'll handle it, don't want to block this on SQS. 🙂